packages/games/src/constants/games/index.ts

```diff
@@ -4,9 +4,10 @@ import { GameType } from '../../types'
 import { binaryX } from './binaryX'
 import { nemesisDownfall } from './nemesisDownfall'
 import { pancakeProtectors } from './pancakeProtectors'
+import { shadowRealm } from './shadowRealm'
 
-export const TOP_GAMES_LIST: GameType[] = [pancakeProtectors]
+export const TOP_GAMES_LIST: GameType[] = [shadowRealm]
 
-export const DEFAULT_GAMES_LIST: GameType[] = [nemesisDownfall, binaryX]
+export const DEFAULT_GAMES_LIST: GameType[] = [nemesisDownfall, binaryX, pancakeProtectors]
```

Comment:
Please change it to.
export const TOP_GAMES_LIST: GameType[] = [pancakeProtectors]
export const DEFAULT_GAMES_LIST: GameType[] = [shadowRealm, nemesisDownfall, binaryX]